dts/bindings/i2c/nuvoton,npcx-i2c-port.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-I2C port pads node
+
+compatible: "nuvoton,npcx-i2c-port"
+
+include: [base.yaml]
+
+properties:
+    label:
+        required: true
+
+    port:
```

Comment:
remove `port` property.   Seems to only be used for LOG_ERR.  Use some other identifier like label or reg instead.

Reply:
Please refer [here](https://github.com/zephyrproject-rtos/zephyr/pull/29144/files#diff-503ccaa7cb43d57ae3e022a9348a1be7c7b78f282a4992da20b6634602fd4ab7R91) The "port" property is important to switch the mux connect to i2c hardware module.